op-chain-ops/genesis/config.go

```diff
@@ -203,10 +203,11 @@ type DeployConfig struct {
 	// FundDevAccounts configures whether or not to fund the dev accounts. Should only be used
 	// during devnet deployments.
 	FundDevAccounts bool `json:"fundDevAccounts"`
-	// The maximum number of participants in the leader election.
-	LeaderElectionMaxParticipants uint64 `json:"leaderElectionMaxParticipants"`
-	// Batcher version at time of deployment
-	InitialBatcherVersion int `json:"initialBatcherVersion"`
+	// The maximum number of leaders in the leader election.
+	LeaderElectionNumberOfLeaders uint64 `json:"leaderElectionNumberOfLeaders"`
+	// The number of consecutive slots assigned to each leader
+	LeaderElectionNumberOfSlotsPerLeader uint64 `json:"leaderElectionNumberOfSlotsPerLeader"`
+	InitialBatcherVersion                int    `json:"initialBatcherVersion"`
```

Comment:
We lost the comment.

```suggestion
	// Batcher version at time of deployment
	InitialBatcherVersion                int    `json:"initialBatcherVersion"`
```